rust/src/s4c25.rs

```diff
@@ -0,0 +1,45 @@
+mod aes;
+
+use aes::MyCiphers;
+use hex::{encode, decode};
+use rand::{Rng, RngCore};
+use std::fs;
+use std::str::from_utf8;
+use std::borrow::BorrowMut;
+
+// This is inefficient. You can set the plaintext to the ciphertext,
+// then CTR will decrypt it for you!
+pub fn crack_random_access_ctr(cipher: &MyCiphers, ct: &mut [u8], nonce: u64) -> Vec<u8> {
+    let mut cracked: Vec<u8> = vec!();
```

Comment:
```suggestion
    let mut cracked = vec!();
```
Type is already inferred from the return type